_includes/head.html

```diff
@@ -6,7 +6,15 @@
     <title>{% if page.title %}{{ page.title }}{% else %}{{ site.title }}{% endif %}</title>
     <meta name="description" content="{{ site.description }}">
 
+    {% if page.canonical_url and page.canonical_url != page.url %}
```

Comment:
Why the condition`page.canonical_url != page.url`? It would be totally valid to annotate a specific URL as the canonical one.

Reply:
So, the logic here is that if a canonical URL is specified and it does not match that of the page, then we're dealing with a redirection page. See the `else` clause immediately below.